ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-ref.yaml

```diff
@@ -81,6 +81,12 @@ ref:
       pool option in install-config.yaml, or when the subnets in Local Zones have been provided. EC2 Instances in Local Zones usually takes
       longer to be ready (Node Ready), so it requires longer time to wait for Node VS Machine count. This flag will extend three times
       than the regular cluster, so it would prevent the monitor to fail before starting the tests.
+  - name: TEST_SDN_LIVE_MIGRATION_OPTIONS
```

Comment:
Dont like adding the definition for the env var here. Have an idea to define it in a cni migration step ref file that will basically do nothing but I dont know yet if the env var will also be passed to subsequent steps (i.e. openshift-e2e step)

Reply:
With the default value being an empty string for this env var, no cni migration will occur.

Reply:
I think it's ok here. this is sort of a common workflow used everywhere but you can see other groups have
followed this same pattern to handle their special cases (e.g., AWS_EDGE_POOL_ENABLED)